examples/example_5.py

```diff
@@ -51,15 +48,18 @@ def example_opt(orca_code, nproc, submit=True):
 
     builder.orca.metadata.options.resources = {
         'num_machines': 1,
-        'num_mpiprocs_per_machine': 1,
+        'num_mpiprocs_per_machine': nproc,
     }
     builder.orca.metadata.options.max_wallclock_seconds = 1 * 10 * 60
     if submit:
-        print('Testing Orca Opt Calculations...')
+        print('Testing basic Orca workflow...')
```

Comment:
@danielhollas maybe just changing this to something like `Testing OrcaBaseWorkChain` for clarity.

Reply:
Thanks, done.